src/main/java/picard/analysis/TheoreticalSensitivityMetrics.java

```diff
@@ -0,0 +1,33 @@
+package picard.analysis;
+
+import htsjdk.samtools.metrics.MetricBase;
+
+/**
+ * Created by fleharty on 6/30/17.
+ */
+public class TheoreticalSensitivityMetrics extends MetricBase {
```

Comment:
@yfarjoun I'm not entirely sure I understand what you mean by histogram.
As I understand it, the histogram stores key-value pairs where the value is a non-negative integer.  I agree with you that we should make this tool so that we can emit sensitivities for any list of allele fractions the user is interested in.